bundler/doc/development/DEBUGGING.md

```diff
@@ -0,0 +1,76 @@
+# Debugging
+
+Debugging Bundler can be challenging, don't be discouraged 🤗.
+
+Make sure you've followed the [development setup](SETUP.md) docs before trying to debug.
+
+## Debugging in tests
+
+Your best option is to print debug in the test suite. Put `puts` statements anywhere you want to see an object or variable and you'll see your `puts` in the console output.
```

Comment:
I mainly use `puts` debugging too! Sometimes printing `caller_locations(0).join("\n")` helps to see what the stacktrace is at a given point.

Reply:
Wonderful, I'll add that to the list of examples